ivy_tests/test_ivy/test_functional/test_core/test_linalg.py

```diff
@@ -1257,6 +1346,50 @@ def test_cholesky(
     )
 
 
+# cov
+@handle_test(
+    fn_tree="functional.ivy.cov",
+    dtype_x1_x2_cov=_get_dtype_value1_value2_cov(
+        available_dtypes=helpers.get_dtypes("numeric"),
+        min_num_dims=1,
+        max_num_dims=2,
+        min_dim_size=2,
+        max_dim_size=5,
+        min_value=-1e10,
```

Comment:
It may be the configuration here:

The test error is:
`RuntimeError: cov(): weights sum to zero, can't be normalized`
(which may be the cause of the ZeroDivisionError)

You can try generating all positive values for example, setting the `min_value` to `0 or 1` (to prevent the probability of generating all 0s too) but keep the `max_value` as it is.